tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -453,3 +453,47 @@ if (/\/2\.9/.test(Cypress.env('rancher_version'))) {
       }
     )
   });
+
+describe('Test application deployment based on clusterGroup', { tags: '@p1'}, () => {
+  qase(25,
+    it("Fleet-25: Test install single application to the all available clusters using 'clusterGroup'", { tags: '@fleet-25' }, () => {
+      const repoName = 'default-single-app-cluster-group-25'
+      const key = 'key_env'
+      const value = 'value_prod'
+      const clusterGroupName = 'cluster-group-env-prod'
+      const dsClusterList = ["imported-0", "imported-1"]
+
+      cy.accesMenuSelection('Continuous Delivery', 'Clusters');
+      cy.contains('.title', 'Clusters').should('be.visible');
+
+      // Assign label to the clusters 
+      dsClusterList.forEach(
+        (dsClusterName) => {
+          cy.assignClusterLabel(dsClusterName, key, value);
+        }
+      )
+
+      // Create group of cluster consists of same label.
+      // dsClusterList[1]: this will provide index 1 element of the array.
+      cy.clickNavMenu(['Cluster Groups']);
+      cy.contains('.title', 'Cluster Groups').should('be.visible');
+      cy.createClusterGroup(clusterGroupName, key, value, dsClusterList[0], 2);
+
+      // Create a GitRepo targeting cluster group created.
+      cy.addFleetGitRepo({ repoName, repoUrl, branch, path, deployToTarget: clusterGroupName });
+      cy.clickButton('Create');
+      cy.checkGitRepoStatus(repoName, '1 / 1');
```

Comment:
Is this going to work on v2.9 where the resources are multiplied by number of clusters?

Reply:
yes, this worked on 2.9. this is bundle count.